docs/index.md

```diff
@@ -108,8 +108,13 @@ snapshot 3671204c saved
 ### Args
 
 * `prune`: A boolean representing whether to automatically run the 'prune' command if snapshots have been removed
+* `keep_last`: An int representing the last N snapshots to keep
+* `keep_hourly`: An int representing the last N hourly snapshots to keep
 * `keep_daily`: An int representing the last N daily snapshots to keep
 * `group_by`: A string for grouping snapshots
```

Comment:
Can we move `group_by` either above or below the `keep_*` so that all the `keep_` args are contiguous?

Reply:
Yep, that makes sense! Done! :)